packages/react-showroom/src/config/create-rspack-config.ts

```diff
@@ -0,0 +1,658 @@
+                  })
+                : undefined,
+              new HtmlWebpackPlugin({
+                filename: '_preview.html',
+                template: resolveShowroom('html-template/preview.html'),
+                templateParameters: {
+                  title: theme.title,
+                  resetCss: theme.resetCss,
+                  prerender: mode === 'production' && !!prerenderConfig,
+                },
+                minify: false,
+                inject: false,
+                chunks: ['preview'],
+              }),
+              html.preview
+                ? new HtmlWebpackTagsPlugin({
+                    ...html.preview,
+                    files: ['**/_preview.html'],
+                  })
+                : undefined,
+            ]),
+        measure
+          ? undefined // don't prettify output message when measuring to avoid important info get cleared
+          : new WebpackMessages({
+              name: 'showroom',
+              logger: logToStdout,
+            }),
+        operation === 'build' && assetDir
+          ? new rspack.CopyRspackPlugin({
+              patterns: [
+                {
+                  from: path.posix.join(assetDir.replace(/\\/g, '/'), '**/*'),
+                  to: resolveApp(outDir),
+                  context: assetDir,
+                  globOptions: {
+                    ignore: ['**/*.html'],
+                  },
+                },
+              ],
+            })
+          : undefined,
+        // TODO no replacement yet (& Workbox is dead too)
+        // theme.serviceWorker && operation === 'build'
+        //   ? new (require('workbox-webpack-plugin').InjectManifest)({
+        //       swSrc: resolveShowroom(
+        //         'client/service-worker/_showroom-service-worker.ts'
+        //       ),
+        //       exclude: [/.wasm$/, /.map$/, /.html$/],
+        //     })
+        //   : undefined,
+      ].filter(isDefined),
+    }),
+    userConfig,
+    mode
+  );
+};
+
+export const createSsrRspackConfig = (
+  mode: Environment,
+  config: NormalizedReactShowroomConfiguration,
+  { outDir = 'showroom', profileWebpack = false, measure = false } = {}
+): rspack.Configuration => {
+  const baseConfig = createBaseRspackConfig(mode, config, {
+    ssr: true,
+    profile: profileWebpack,
+    measure,
+    operation: 'build',
+  });
+
+  const showroomServer = resolveShowroom(
+    'client-dist/app/showroom-server-entry.js'
+  );
+  const previewServer = resolveShowroom(
+    'client-dist/app/preview-server-entry.js'
+  );
+
+  return mergeWebpackConfig(
+    // FIXME
+    // @ts-ignore-errors
+    merge(baseConfig, {
+      entry: {
+        ...(config.require ? { requireConfig: config.require } : {}),
+        prerender: showroomServer,
+        previewPrerender: previewServer,
+      },
+      output: {
+        path: resolveApp(`${outDir}/server`),
+        filename: '[name].js',
+        chunkFilename: '[name].js',
```

Comment:
This is a workaround bc `runtimeChunk` was not named according to `chunkFilename` but `filename`